test/sslapitest.c

```diff
@@ -134,6 +142,50 @@ struct sslapitest_log_counts {
     unsigned int exporter_secret_count;
 };
 
+#ifndef OSSL_NO_USABLE_ECH
+
+# define OSSL_ECH_MAX_LINELEN 1000 /* for a sanity check */
+
+static char *echconfiglist_from_PEM(const char *echkeyfile)
```

Comment:
You may add a second parameter to return the `readbytes` value.

Reply:
Could do, but as it returns a string, using ``strlen()`` on that seems ok, esp. for test code.